roles/download/defaults/main.yml

```diff
@@ -512,6 +515,15 @@ downloads:
     tag: "{{ test_image_tag }}"
     sha256: "{{ testbox_digest_checksum|default(None) }}"
 
+  dashboard:
+    enabled: "{{ dashboard_enabled }}"
+    container: true
+    repo: "{{ dashboard_image_repo }}"
+    tag: "{{ dashboard_image_tag }}"
+    sha256: "{{ dashboard_digest_checksum|default(None) }}"
+    groups:
+      - kube-node
```

Comment:
Dashboard will run on Masters too.

Reply:
Dashboard image will not download and sync in "pull once, push many" mode without this variable declare

Reply:
please view this #3030

Reply:
i mean set this Dashboard image will be pull in deploy node and than save in tar on deploy node, deploy node will send Dashboard image tar file to k8s master and work node . in k8s master and worker node ,it does not pull images form internet , it will use tar file to load image . with out set this variable, it will miss Dashboard image to translate.